babel.config.js

```diff
@@ -5,25 +5,31 @@ const plugins = [
   '@babel/plugin-proposal-export-namespace-from',
   ['@babel/plugin-proposal-decorators', { legacy: true }],
   ['@babel/plugin-proposal-class-properties', { loose: true }],
+  ['@babel/plugin-proposal-private-property-in-object', { loose: true }],
+  ['@babel/plugin-proposal-private-methods', { loose: true }],
   '@babel/plugin-proposal-object-rest-spread',
```

Comment:
during tests I have a lots of logs complaining about loose option to be also added to those two babel plugins.